src/Components/Countries/Countries.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGlobe } from '@fortawesome/free-solid-svg-icons';
+import { getCountries } from '../../Redux/ReducerCountries';
+import Country from './Contry';
+import '../../index.css';
+
+const Countries = () => {
+  const [search, setSearch] = useState([]);
+  const dispatch = useDispatch();
+  const { loading, countries } = useSelector((state) => state.countriesReducer);
+
+  useEffect(() => {
+    dispatch(getCountries());
+  }, [dispatch]);
+
+  useEffect(() => {
+    setSearch(countries);
+  }, [countries]);
+
+  const SearchHandler = (e) => {
+    setSearch(
+      countries.filter((c) => c.id.toLowerCase().includes(e.target.value)),
+    );
+  };
+  return (
+    <div className="container">
+      <div className="row">
+        <div className="d-flex col col-6">
+          <FontAwesomeIcon icon={faGlobe} className="border-black my-3 W-100" />
+        </div>
+        <div className="align-items-center col col-6 font-weight-bold">
+          <h1 className="">TRACK COVID DATA HERE</h1>
+          <h2 className="mt-3">All Countries</h2>
+          <h3>
+            Countries:
+            {countries.length}
+          </h3>
+        </div>
+        <div>
+          <input type="text" placeholder="STATS BY COUNTRY" onChange={SearchHandler} className="p-2 search-country" />
```

Comment:
- [ ] It would be better if our search filter functionality works as expected. Currently, when entering the names of the country, data result is not returned. Let us kindly fix it